js/src/carousel.js

```diff
@@ -23,7 +23,7 @@ const JQUERY_NO_CONFLICT     = $.fn[NAME]
 const ARROW_LEFT_KEYCODE     = 37 // KeyboardEvent.which value for left arrow key
 const ARROW_RIGHT_KEYCODE    = 39 // KeyboardEvent.which value for right arrow key
 const TOUCHEVENT_COMPAT_WAIT = 500 // Time for mouse compat events to fire after touch
-const SWIPE_THRESHOLD        = 40
+const SWIPE_THRESHOLD        = 40 // Minimum swipe displacement in unit of DOM coordinate, usually in "px"
```

Comment:
I'm not sure this is better, personally.

Reply:
Would you mind suggest how to improve it?

Reply:
I'm not a native English speaker either, so bear with me :P

But maybe, just `Minimum swipe distance`. Although I'm still not sure if it adds any value then.

Reply:
Well, the reason I added the doc is when I was looking into the code and was not sure what unit the number is, really I was trying to simply say that this number is in the unit of pixel rather than anything else. Do you think it is clearer by just shorten it to `In the unit of \`px\``?